domains/atomu.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "atomu21263",
+        "email": "atom.aizu@gmail.com"
+      },
+
+      "record": {
+        "A": ["106.73.4.128"]
```

Comment:
404?

Reply:
use port :8816
(my HGW doesnt open 80,433

Reply:
@atomu21263 Sorry, your website needs to be in English.

Reply:
hmm other my web service :29300?
source: https://github.com/atomu21263/paint

Reply:
@atomu21263 That will work, but you need to add a main page.

Reply:
main page URL is ...4.128:430/?

Reply:
> main page URL is "...4.128:430/"?

Oh. It needs to be on the same IP, I believe.

Reply:
what? (i'm not good english,sorry.
my home Router(?) is doesnt open 80,433 and other Well Known Ports...

Reply:
> what? (i'm not good english,sorry.
> my home Router(?) is doesnt open 80,433 and other Well Known Ports...

@atomu21263 I would highly recommend, **not** using your router for your ip.

Reply:
hmm how to do? or what search keyword?

Reply:
You should change your IP immediately since you'll probably get DDOSed

Reply:
I can't change it because of my IPv6 contract, so can you delete this Pull request?

Reply:
You cannot delete PRs. Only close them, so I recommend closing this PR and hope that no one DDOSes you.

Reply:
ok, thank you support.
pls close PRs after check this message

Reply:
I can't close PR's. You have to. Use the close PR button at the bottom of the page.

Reply:
ok, thx.